docs/resources/service.md

```diff
@@ -41,12 +41,12 @@ The following arguments are supported:
 * `team` - (Optional) Variable number of blocks containing team assignments for this service.
 	* `role` - (Optional) The service's role in the team. If defined, must be one of `Member` or `Manager`.
 	* `slug` - (Required) The team the service should be added to.
-
+* `return_api_key` - (Optional) The service's API key to be returned in state. Defaults to `true`. If se to `false`, the "key" value is replaced with `disabled`
```

Comment:
```suggestion
* `return_api_key` - (Optional) The service's API key to be returned in state. Defaults to `true`. If set to `false`, the "key" value is replaced with `disabled`
```

Reply:
Maybe also worth being explicit, with a string like `**redacted**` rather than disabled, might be a smidge more clear?